google/shared/instance_group_manager.go

```diff
@@ -0,0 +1,156 @@
+	InstanceGroup string `json:"instanceGroup,omitempty"`
+
+	// InstanceTemplate: The URL of the instance template that is specified
+	// for this managed instance group. The group uses this template to
+	// create all new instances in the managed instance group.
+	InstanceTemplate string `json:"instanceTemplate,omitempty"`
+
+	// Kind: [Output Only] The resource type, which is always
+	// compute#instanceGroupManager for managed instance groups.
+	Kind string `json:"kind,omitempty"`
+
+	// Name: The name of the managed instance group. The name must be 1-63
+	// characters long, and comply with RFC1035.
+	Name string `json:"name,omitempty"`
+
+	// NamedPorts: Named ports configured for the Instance Groups
+	// complementary to this Instance Group Manager.
+	NamedPorts []*NamedPort `json:"namedPorts,omitempty"`
+
+	// Region: [Output Only] The URL of the region where the managed
+	// instance group resides (for regional resources).
+	Region string `json:"region,omitempty"`
+
+	// SelfLink: [Output Only] The URL for this managed instance group. The
+	// server defines this URL.
+	SelfLink string `json:"selfLink,omitempty"`
+
+	// TargetPools: The URLs for all TargetPool resources to which instances
+	// in the instanceGroup field are added. The target pools automatically
+	// apply to all of the instances in the managed instance group.
+	TargetPools []string `json:"targetPools,omitempty"`
+
+	// TargetSize: The target number of running instances for this managed
+	// instance group. Deleting or abandoning instances reduces this number.
+	// Resizing the group changes this number.
+	TargetSize int64 `json:"targetSize,omitempty"`
+
+	// Zone: [Output Only] The URL of the zone where the managed instance
+	// group is located (for zonal resources).
+	Zone string `json:"zone,omitempty"`
+
+	// ServerResponse contains the HTTP response code and headers from the
+	// server.
+	googleapi.ServerResponse `json:"-"`
+
+	// ForceSendFields is a list of field names (e.g. "BaseInstanceName") to
+	// unconditionally include in API requests. By default, fields with
+	// empty values are omitted from API requests. However, any non-pointer,
+	// non-interface field appearing in ForceSendFields will be sent to the
+	// server regardless of whether the field is empty or not. This may be
+	// used to include empty fields in Patch requests.
+	ForceSendFields []string `json:"-"`
+
+	// NullFields is a list of field names (e.g. "BaseInstanceName") to
+	// include in API requests with the JSON null value. By default, fields
+	// with empty values are omitted from API requests. However, any field
+	// with an empty value appearing in NullFields will be sent to the
+	// server as null. It is an error if a field in this list has a
+	// non-empty value. This may be used to include null fields in Patch
+	// requests.
+	NullFields []string `json:"-"`
+}
+
+func (s *InstanceGroupManager) ToProduction() *compute.InstanceGroupManager {
```

Comment:
Let's swap out `Production` with `v1` everywhere, since there could be more than one production version.